src/core/resources/search/tokenSearch.ts

```diff
@@ -178,6 +193,7 @@ export function useTokenSearch(
     }),
     queryFn: tokenSearchQueryFunction,
     ...config,
+    placeholderData: (previousData) => previousData,
```

Comment:
this is the default behavior so this does not do anything

Reply:
BX is on react query v5, `placeholderData: (previousData) => previousData,` is the same as the old `keepPreviousData`
https://tanstack.com/query/v5/docs/framework/react/guides/migrating-to-v5#removed-keeppreviousdata-in-favor-of-placeholderdata-identity-function